spec/music_album_spec.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../src/music_album'
+
+describe 'Testing music_album' do
+  context 'Creating a new music_album ' do
+    new_music_album = MusicAlbum.new(true, 'Publisher1', 'Fiction', 'Author', 'Gift', '1990-11-11')
```

Comment:
Kindly remove `Publisher` from the arguments because we don't have it in the main class. ✅
![image](https://user-images.githubusercontent.com/107952724/213030314-09d83320-811e-49a5-aa89-8e60b3cd7e37.png)